accel-pppd/triton/triton.c

```diff
@@ -55,7 +55,7 @@ static __thread struct triton_context_t *this_ctx;
 static __thread jmp_buf jmp_env;
 static __thread void *thread_frame;
 
-#define log_debug2(fmt, ...)
+//#define log_debug2(fmt, ...)
```

Comment:
this will be quite noisy and will have performance impact, why do you want to make it default for everyone?

Reply:
I found that with the define in place there were no thread/context massages in debug log at all. Looks a bit missleading to the part from doc in that regard: "...also include mikrotime and threads numbers..."

Reply:
reverted